<?xml version="1.0"?>

<launch>
    <node name="enif_iuc_ground" pkg="enif_iuc" type="enif_iuc_ground" respawn="true" launch-prefix="" output="screen">
      <param name="USB" type="string" value="/dev/xbee"/>
    </node>
</launch>
